src/reducers/homeReducer.js

```diff
@@ -0,0 +1,75 @@
+import actions from '../actions/index';
+const initialState = {
+ bannerData: [],
+ bannerdata_searching_failed: false,
+ bannerdata_searching_success: false,
+ bannerdata_searching: false,
+ categoryData: [],
+ categorydata_searching_failed: false,
+ categorydata_searching_success: false,
+ categorydata_searching: false
```

Comment:
same as above

Reply:
No, I want to keep it independent for every api call here.